src/utils/urls.js

```diff
@@ -157,8 +157,21 @@ function getLessonUrl({ id, slug, view, templateLessonId }) {
   return queryString ? `${url}?${queryString}` : url;
 }
 
+function getDocIdIfDocUrl(url) {
+  const docPageRegex = new RegExp(`^(?:${docsPrefix})([a-zA-Z0-9]+)\\b`, 'i');
```

Comment:
We could extract those two regexes into a module level variable to have them closer to where also `docsPrefix` and `lessonsPrefix` live and be created just once (as they're called for each request). Also, maybe, although we know that `docsPrefix` and `lessonsPrefix` at the moment do not contain characters that would have to be escaped for use in a regex, it wouldn't hurt escaping them anyways.

Reply:
Good point, extracted them both to module level and escaped prefixes.